packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -553,6 +580,46 @@ class EditorPage {
 	async sauceJobStatus( allPassed ) {
 		await this.driver.sauceJobStatus( allPassed );
 	}
+
+	// =============================
+	// Misc functions
+	// =============================
+	async findElement( name, iosElementType ) {
+		const elementName = isAndroid()
+			? '//*'
+			: `//XCUIElementType${ iosElementType }`;
+		const blockLocator = `${ elementName }[contains(@${ this.accessibilityIdXPathAttrib }, "${ name }")]`;
+		const elements = await this.driver.elementsByXPath( blockLocator );
+		return elements[ 0 ];
+	}
+
+	async selectElement( name, iosElementType ) {
+		const el = await this.findElement( name, iosElementType );
+		el.click();
+	}
+
+	async selectBlock(
+		blockName,
+		position = 1,
+		options = { autoscroll: false }
+	) {
+		const block = await this.getBlockAtPosition(
+			blockName,
+			position,
+			options
+		);
+		block.click();
+	}
+
+	async tapCoordinates( x, y, longPress = false ) {
```

Comment:
If possible, using `clickMiddleOfElement` instead could be more robust. The device size, running locally or in the cloud, could be different and coordinates may not be the same between them.

Reply:
Hmmm. The problem is that I need to click outside of the alert and I can't get elements behind the alert.

Reply:
Would pressing the back button instead on Android work?